docs/parameters.rst

```diff
@@ -2,58 +2,256 @@
+    dtype: str
+    example: "Survived"
 
-- is_combined(``bool``): Parameter to determine whether columns should be scaled together as a group
+- **cat_cols**
 
-- threshold(``dict``): Dictionary of threshold values where the key is the column name and the value is the threshold for that column.
+List of column names provided by user indicating these columns are to be encoded categorically. If ``None`` then ``Preprocessy`` calculates and identifies the columns on its own.
 
-- score_func(``callable``): Function taking two arrays X and y, and returning a pair of arrays
-                     ``(scores, pvalues)`` or a single array with scores. ``score_func`` can be custom
-                     or used from ``sklearn.feature_selection``
+.. code:: python
 
-- k(``int``): Number of top features to select.
+    dtype: list[str]
+    example: ["Gender", "Cabin", "Embarked"]
 
-- test_size(``float,int``): Size of test set after splitting. Can take values from 0 - 1 for float point values, 0 - Number of samples for integer values. Is complementary to train size.
+- **ord_dict**
 
-- train_size(``float,int``): Size of train set after splitting. Can take values from 0 - 1 for float point values, 0 - Number of samples for integer values. Is complementary to test size.
+Ordinal attributes require an associated weight mapping. This parameter is mapping from attribute column
+name to weight mapping.
 
-- n_splits(``int``): Number of folds to be made in K-fold cross validation. Must be at least 2. For more see :py:mod:`preprocessy.data_splitting.KFold`
+.. code:: python
 
-- shuffle(``bool``): Decides whether to shuffle data before splitting.
+    dtype: dict[str => dict[str => int | float]]
+    example: {
+        "Difficulty": {
+            "Easy": 5
+            "Medium": 10,
+            "Hard": 15
+        }
+    }
 
-- random_state(``int``): Seeding to be provided for shuffling before splitting.
+- **replace_cat_nulls**
+
+When handling missing data for categorical and ordinal attributes, the ``int`` provided here will be used
+to replace the null value. If no value is provided, then the column will be dropped.
+
+.. code:: python
+
+    dtype: int
+    example: 99
+
+- **drop_cols**
+
+List of column names to be dropped.
+
+.. code:: python
+
+    dtype: list[str]
+    example: ["PassengerId", "Name"]
+
+- **fill_missing**
+
+Dictionary of format {"method": [col]} to indicate the method (``mean``/``median``) to be applied on specified list of columns.
+
+.. code:: python
+
+    dtype: dict["mean" | "median" => list[str]]
+    example: {
+        "mean": ["col_A", "col_B"],
+        "median": ["col_C"]
+    }
+
+- **fill_values**
+
+Dictionary with keys as column names and values that fill the null records in corresponding column.
+
+.. code:: python
+
+    dtype: dict[str => any]
+    example: {
+        "Age": 19,
+        "Name": "John"
+    }
+
+- **one_hot**
+
+``True`` if one hot encoding desired. Default = ``False``.
```

Comment:
one hot encoding is* desired

Reply:
Since a default value is provided, this is not a required param

Reply:
Yeah I mean word "is" is missing.

Reply:
Oh typo